sdk/sdk.go

```diff
@@ -17,6 +18,8 @@ import (
 
 // SDK exposes all functionalities of MESG core.
 type SDK struct {
+	ServiceSDK *servicesdk.Service
```

Comment:
rename in `Service` we are already in the `SDK` so we can access with `sdk.Service`

Reply:
done